src/lib/get-session-id.js

```diff
@@ -0,0 +1,45 @@
+// const stored = sessionStorage ? sessionStorage.getItem('session_id') : null;
```

Comment:
Since this whole file is commented out, is this file a placeholder?

Reply:
Yes, I left it as a placeholder because I felt this work was most appropriately done for a future PR rather than making this PR too fat.